styles/styles.css

```diff
@@ -33,6 +33,73 @@
   --highlight-background-color: #ccc;
   --text-color: #000;
 
+  /* Netcentric colors */
```

Comment:
you should include the Netcentric colors first, and then update the values for the existing used colors (link, link hover, text, etc.)